src/jit/rationalize.cpp

```diff
@@ -1974,7 +1975,10 @@ Compiler::fgWalkResult Rationalizer::SimpleTransformHelper(GenTree **ppTree, Com
             break;
 
         case GT_LDOBJ:
-            RewriteLdObj(ppTree, data);
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
+            if (varTypeIsSIMD(tree))
```

Comment:
I think even on Windows Amd64 we expect to see LdObj only on simd Types.  Hence there is no need to enclose if condition inside #ifdef.